server/src/middleware/routes.test.ts

```diff
@@ -1275,4 +1255,118 @@ describe("Endpoints", () => {
       expect(res.status).toEqual(401)
     })
   })
+
+  describe("/bookings/create-bookings", () => {
+    beforeEach(async () => {
+      await createUsers()
+    })
+
+    afterEach(async () => {
+      await cleanFirestore()
+    })
+
+    it("should return userIds for successful bookings within the date range", async () => {
```

Comment:
we should add a check to directly query firestore and see if the bookings are actually being created properly